packages/contracts/contracts/plugin/PluginInstaller.sol

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: MIT
+
+import "@openzeppelin/contracts/utils/introspection/ERC165Checker.sol";
+
+import {Permission, PluginManager} from "./PluginManager.sol";
+import {PluginERC1967Proxy} from "../utils/PluginERC1967Proxy.sol";
+import {AragonUpgradablePlugin} from "../core/plugin/AragonUpgradablePlugin.sol";
+import {AragonPlugin} from "../core/plugin/AragonPlugin.sol";
+import {DAO} from "../core/DAO.sol";
+
+/// @notice Plugin Installer that has root permissions to install plugin on the dao and apply permissions.
+contract PluginInstaller {
```

Comment:
Because we have `install`, `update`, and `uninstall`, we should consider renaming `PluginInstaller` to something like `PluginSetupExecutor` or just `SetupExecutor` or similar.
I would also rename `PluginManager` to `PluginSetupInstruction` or just `Setup`.